pyzo/core/main.py

```diff
@@ -252,19 +252,21 @@ def saveWindowState(self):
         # Store window geometry
         geometry = self.saveGeometry()
         try:
-            geometry = bytes(geometry)  # PyQt4
+            #geometry = bytes(geometry)  # PyQt4
+            pass
         except Exception:
             geometry = bytes().join(geometry)  # PySide
-        geometry = base64.encodebytes(geometry).decode("ascii")
+        geometry = base64.encodebytes(geometry.data()).decode("ascii")
         pyzo.config.state.windowGeometry = geometry
```

Comment:
So this makes the try-except empty, because there is a single approach for both PyQt5 and PySide2, right? Lets remove the try-except too then, please :)

Reply:
Hi, so unfortunately I do not have sufficient knowledge to cleanly solve this issue. I thing someone with deeper knowledget of PyQt5/PySide2 should look at this.
With this "temporar" solution at least it does not Segmentation Fault.

Reply:
Ok, no problem. In that case I'll merge and do a bit of cleanup directly afterwards.